contracts/CO30_Emerson_UncontrolledDet.SefrouRamal/edit_me/briefing.sqf

```diff
@@ -0,0 +1,87 @@
+#include "..\do_not_edit\script_component.hpp"
+/*
+ * Author: Kresky, Jonpas
+ * Adds briefing based on side.
+ * Call from initPlayerLocal.sqf.
+ * Copy to other side briefings if you want to add briefings to those sides.
+ *
+ * Arguments:
+ * 0: Player <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player] call FUNC(briefing);
+ *
+ * New Line:
+ * <br/>
+ *
+ * Text Colour (Hex colour Code)
+ * <font color='#FF0000'>TEXT HERE</font>
+ *
+ * Clickable Marker Link:
+ * <marker name = 'MARKER_NAME'>MARKER_TEXT</marker>
+ *
+ * Image:
+ * <img image='FOLDER\IMAGE.jpg' width='200' height='100'/>
+ */
+
+params ["_player"];
+
+switch (side _player) do {
+    // BLUFOR briefing
+    case west: {
+        
+    };
+
+    // OPFOR briefing
+    case east: {
+
+    };
+
+    // RESISTANCE/INDEPENDENT briefing -- THIS IS THE BRIEFING YOU WANT TO EDIT FOR YOUR MISSION!
+    case resistance: {
+        _player createDiaryRecord ["Diary", ["Mission", "
+            <br/>Contract Name: Uncontrolled Det
+            <br/>Local Time: 1600
+            <br/>Local Weather: Sunny
```

Comment:
maybe not a huge fan of this since we've had some people be unreasonable about anything with the sun within 10º of the horizon

Reply:
To clarify, I don't know how I feel about saying the time in the brief, I'm fine with the actual time of your mission

Reply:
Ah ok. I can remove it, though I figured it would be useful to put starting time in briefing if we are trying to get more organized with preselecting OLs and maybe having them preplan before session time.